markdown/landing/index.md

```diff
@@ -20,6 +20,8 @@ All documentation and media hosted is licensed under CC-BY-SA 4.0, with no exclu
 
 At this current moment, no marked-stable builds of Nazi Zombies: Portable are made available. Instead, nightly builds on all platforms are released via GitHub releases, accessible [here](https://github.com/nzp-team/nzportable/releases/latest). This link will always take you to the latest release, identifiable via the build date in the release title.
 
+If you wish to setup a Dedicated Server for Nazi Zombies: Portable, [click here](../internal/dedicated-server.md)
```

Comment:
Perhaps being an item under "Core Documentation" would be a better place for this? I can revise the description for this section to make it feel more in-place, but floating on its own doesn't feel right.

Reply:
I thought it was related to that section but can move it to core documentation, putting it under the list with Weapon IDs